src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs

```diff
@@ -2,12 +2,16 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 // This program uses code hyperlinks available as part of the HyperAddin Visual Studio plug-in.
 // It is available from http://www.codeplex.com/hyperAddin 
+#if !PLATFORM_UNIX
+
 #define FEATURE_MANAGED_ETW
```

Comment:
Given my feedback for the FeatureManagedEtw above, this worries me.  I would consider using something like ```#if !INSIDE_CLR``` so when we are in mscorlib we just use whatever is in the props file.